src/backend/tools.ts

```diff
@@ -352,7 +358,8 @@ export const Winetricks = {
       const linuxEnvs = {
         ...process.env,
         WINEPREFIX: winePrefix,
-        PATH: `${winepath}:${process.env.PATH}`
+        PATH: `${winepath}:${process.env.PATH}`,
+        ...setupWineEnvVars(gameSettings, appName)
```

Comment:
Now that you're adding `setupWineEnvVars`, shouldn't we also have `setupEnvVars` here?